src/test/java/no/digipost/monitoring/event/AppBusinessEventLoggerTest.java

```diff
@@ -40,40 +40,40 @@ void setUp() {
+        assertThat(prometheusRegistry.scrape(), containsString("app_business_events_total{name=\"VIOLATION\"} 1.0"));
         assertThat(prometheusRegistry.scrape(), not(containsString("app_business_events_1min_warn_thresholds")));
         assertThat(prometheusRegistry.scrape(), not(containsString("app_business_events_1min_error_thresholds")));
     }
 
     @Test
     void should_register_event_with_warn() {
         eventLogger.log(MyBusinessEvents.VIOLATION_WITH_WARN);
-        assertThat(prometheusRegistry.scrape(), containsString("app_business_events_total{name=\"VIOLATION_WITH_WARN\",} 1.0"));
-        assertThat(prometheusRegistry.scrape(), containsString("app_business_events_1min_warn_thresholds{name=\"VIOLATION_WITH_WARN\",} 5.0"));
+        assertThat(prometheusRegistry.scrape(), containsString("app_business_events_total{name=\"VIOLATION_WITH_WARN\"} 1.0"));
+        assertThat(prometheusRegistry.scrape(), containsString("app_business_events_1min_warn_thresholds{name=\"VIOLATION_WITH_WARN\"} 5.0"));
         assertThat(prometheusRegistry.scrape(), not(containsString("app_business_events_1min_error_thresholds")));
     }
 
     @Test
     void should_register_event_with_error() {
         eventLogger.log(MyBusinessEvents.VIOLATION_WITH_ERROR);
-        assertThat(prometheusRegistry.scrape(), containsString("app_business_events_total{name=\"VIOLATION_WITH_ERROR\",} 1.0"));
+        assertThat(prometheusRegistry.scrape(), containsString("app_business_events_total{name=\"VIOLATION_WITH_ERROR\"} 1.0"));
         assertThat(prometheusRegistry.scrape(), not(containsString("app_business_events_1min_warn_thresholds")));
-        assertThat(prometheusRegistry.scrape(), containsString("app_business_events_1min_error_thresholds{name=\"VIOLATION_WITH_ERROR\",} 5.0"));
+        assertThat(prometheusRegistry.scrape(), containsString("app_business_events_1min_error_thresholds{name=\"VIOLATION_WITH_ERROR\"} 5.0"));
     }
 
     @Test
     void should_register_event_with_warn_and_error() {
         eventLogger.log(MyBusinessEvents.VIOLATION_WITH_WARN_AND_ERROR);
-        assertThat(prometheusRegistry.scrape(), containsString("app_business_events_total{name=\"VIOLATION_WITH_WARN_AND_ERROR\",} 1.0"));
-        assertThat(prometheusRegistry.scrape(), containsString("app_business_events_1min_warn_thresholds{name=\"VIOLATION_WITH_WARN_AND_ERROR\",} 5.0"));
-        assertThat(prometheusRegistry.scrape(), containsString("app_business_events_1min_error_thresholds{name=\"VIOLATION_WITH_WARN_AND_ERROR\",} 5.0"));
+        assertThat(prometheusRegistry.scrape(), containsString("app_business_events_total{name=\"VIOLATION_WITH_WARN_AND_ERROR\"} 1.0"));
+        assertThat(prometheusRegistry.scrape(), containsString("app_business_events_1min_warn_thresholds{name=\"VIOLATION_WITH_WARN_AND_ERROR\"} 5.0"));
+        assertThat(prometheusRegistry.scrape(), containsString("app_business_events_1min_error_thresholds{name=\"VIOLATION_WITH_WARN_AND_ERROR\"} 5.0"));
```

Comment:
The output now seems to exclude the trailing comma after upgrading to Micrometer 1.13.0.